packages/cli/src/commands/test.ts

```diff
@@ -55,7 +56,7 @@ export default class Test extends Command {
 
   async run() {
     const { args } = await this.parse(Test)
-    const basePath = args.path ?? process.cwd()
+    const basePath = args.path ? path.resolve(args.path) : process.cwd()
```

Comment:
This code is repeated in many files, maybe creating a pure function would be helpful. 

```suggestion
    const basePath = getBasePath(args.path)
```

```ts
function getBasePath(path: string){
  if(path) {
    return path.resolve(path)
  }

  return process.cwd()
}
```